specification/attestation/resource-manager/Microsoft.Attestation/attestation.json

```diff
@@ -0,0 +1,429 @@
+                        "description": "Get the status of attestation Provider",
+                        "schema": {
+                            "$ref": "#/definitions/AttestationProvider"
+                        }
+                    },
+                    "404": {
+                        "description": "Attestation provider not found"
+                    },
+                    "default": {
+                        "description": "Error result from Attestation service",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResult"
+                        }
+                    }
+                }
+            },
+            "put": {
+                "tags": [
+                    "AttestationProvider"
+                ],
+                "operationId": "AttestationProviders_Create",
+                "description": "Creates or updates the Attestation Provider.",
+                "x-ms-examples": {
+                    "AttestationProviders_Create": {
+                        "$ref": "./examples/Create_AttestationProvider.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "name": "providerName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "pattern": "^[a-zA-Z0-9-]{3,24}$",
+                        "description": "Name of the attestation service"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "creationParams",
+                        "in": "body",
+                        "description": "Client supplied parameters.",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/AttestationServiceCreationParams"
+                        },
+                        "x-ms-parameter-location": "method"  
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Created or updated attestation service",
+                        "schema": {
+                            "$ref": "#/definitions/AttestationProvider"
+                        }
+                    },
+                    "201": {
+                        "description": "Created or updated attestation service",
+                        "schema": {
+                            "$ref": "#/definitions/AttestationProvider"
+                        }
+                    },
+                    "409": {
```

Comment:
The 409 doesn't need to be modeled, but how would the client get a 409? PUT methods are create or update, so a PUT to the same resourceId should be an update, not a conflict. Is there another restriction on properties of the resource that could cause conflict with discrete resourceIds?

Reply:
removed 
                    "409": {
                        "description": "Duplicate attestation name.",
                        "schema": {
                            "$ref": "#/definitions/ErrorResponse"
                        }
                    },
We don't provide update function